guptahemant/insta-clone
# #29 - Added a story block and story page on the site

Requirement: A story block as per the original site to be made with all the users picture and user information. and a page with all the stories contained by the users flagged entity.

Testing Steps: 
1. Take the fresh pull of the branch
2. composer install / Drush cim download and install all modules necessary
3. create a story from any user a block will popup once story is added.
4. Add minimum 5 story to check it works.
5. click on the user profile image from the block
6. you can view the story.

![Screenshot from 2022-04-21 11-47-58](https://user-images.githubusercontent.com/87809169/164386869-958125ad-a7c3-4a47-8637-b4262d96843d.png)
![Screenshot from 2022-04-21 11-49-08](https://user-images.githubusercontent.com/87809169/164387030-730e1a10-781b-4429-8353-17e93660f347.png)
